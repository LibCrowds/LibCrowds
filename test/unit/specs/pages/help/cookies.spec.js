import BootstrapVue from 'bootstrap-vue'

import { mount, createLocalVue } from '@vue/test-utils'
import CookiesHelp from '@/pages/help/cookies'

describe('API help page', () => {
  let localVue = null
  let wrapper = null

  beforeEach(() => {
    localVue = createLocalVue()
    localVue.use(BootstrapVue)
    wrapper = mount(CookiesHelp, {
      localVue
    })
  })

  it('renders correctly', () => {
    const renderer = require('vue-server-renderer').createRenderer()
    renderer.renderToString(wrapper.vm, (err, str) => {
      expect(str).toMatchSnapshot()
    })
  })
})
