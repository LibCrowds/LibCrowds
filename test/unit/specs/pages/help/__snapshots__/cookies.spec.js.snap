// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`API help page renders correctly 1`] = `
<div class="card card-base">
  <div class="card-header">
    <div class="row mb-0 d-flex align-items-center">
      <div class="col">
        <h6 class="mb-0">Cookies Policy</h6>
        <p class="text-muted mb-0">
          <small>
            The My Brand Cookies Policy.
          </small>
        </p>
      </div>
      <!---->
    </div>
  </div>
  <!---->
  <div class="card-body">
    <p>
      My Brand uses cookies to make our website easier for you to use and improve your overall experience, distinguish you from other users and provide increased functionality.
    </p>
    <h3 class="mt-3">What are cookies?</h3>
    <p>
      Cookies are small text files that websites save to your computer. They often include a randomly generated number which is stored on your device. Many cookies are automatically deleted after you finish using the website. Others remain on your computer
      to provide a seamless user experience â€“ for example, by remembering which goods in an online shopping basket.
    </p>
    <p>
      Cookies are commonly used to improve the browsing experience; measure website performance; support the delivery of services; and support sharing information through social media platforms, such as Twitter or Facebook.
    </p>
    <p>
      Unless you have adjusted your browser settings to refuse cookies, our systems will issue cookies as soon you visit our website or access other online services. If you have switched off cookies then some of the functionality of our services may not be
      available to you.
    </p>
    <p>
      The following video from Google gives an explanation of how cookies work:
    </p>
    <div class="text-center my-3">
      <iframe width="640" height="386" src="https://www.youtube.com/embed/TBR-xtJVq7E" frameborder="0" allowfullscreen="allowfullscreen"></iframe>
    </div>
    <h3 class="mt-3">Disabling cookies</h3>
    <p>
      You can block cookies by activating the setting on your browser that allows you to refuse all or some cookies. However, if you use your browser settings to block all cookies (including essential cookies) you may not be able to access parts of our website,
      or you may experience reduced functionality when accessing certain services. Unless you have adjusted your browser setting so that it will refuse cookies, our system will issue cookies as soon you visit our website.
    </p>
  </div>
</div>
`;
